### PR TITLE
feat(test_apps): Add noglib test_app

### PR DESCRIPTION
**Pull request by tore-espressif**
# ESP-BSP Pull Request checklist

> Note: For new BSPs create a PR with this [link](https://github.com/espressif/esp-bsp/compare/main...my-branch?quick_pull=1&template=pr_template_bsp.md).
- [x] Version of modified component bumped
- [x] CI passing
- [x] **Add README!!!**: Full noglib documentation will be added together with noglib CI refactoring

- Follow up in https://github.com/espressif/esp-bsp/pull/384

# Change description
* This example is based on https://github.com/espressif/esp-idf/tree/master/examples/peripherals/lcd/tjpgd
* To run it locally, you have to run `python ./.github/ci/bsp_noglib.py bsp/esp_wrover_kit` !

### CODE REVIEW

**Comment by tore-espressif** [comment]
@espzav @pborcin This is ready for review. It is only a simple noglib example, which is not yet build in CI.

In final follow-up PR I want to update the docs, build this example in CI and push created noglib BSP version to release/noglib branch

**Review by espzav — COMMENTED** [review]
@tore-espressif Thank you for this example. It is ok, left one comment about rotation. I think, we should find common solution.

File: examples/noglib/main/noglib_main.c [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    // Allocate memory for the pixel buffers
+    for (int i = 0; i < FRAME_BUF_NUM; i++) {
+        s_lines[i] = heap_caps_malloc(FRAME_BUF_SIZE, MALLOC_CAP_DMA);
+        assert(s_lines[i] != NULL);
+    }
+
+    // Start and rotate
+    while (1) {
+        // Set driver configuration to rotate 180 degrees each time
+        ESP_ERROR_CHECK(esp_lcd_panel_mirror(lcd_panel, is_rotated, is_rotated));
```
[the remark this answers is not in the document]
Comment (espzav) [reply examples/noglib/main/noglib_main.c:83]: I tried it. It put right colors, but the screen is still mirrored.

**Comment by tore-espressif** [comment]
@espzav I moved the 'example' to test_apps/noglib
The original test_app was moved to test_apps/components

PTAL!

File: .idf_build_apps.toml [hunk 2]
```diff
@@ -1,11 +1,5 @@
 target = "all"
-paths = "."
 recursive = true
-exclude = [
```
[the remark this answers is not in the document]
Comment (tore-espressif) [reply .idf_build_apps.toml:4]: Added back

File: examples/noglib/main/noglib_main.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (tore-espressif) [reply examples/noglib/main/noglib_main.c]: @espzav In this PR we have the new example + refactored CI.
I'd like to leave this issue and extension to all noglib BSPs for a follow-up PR. Please, let me know if you agree

[the remark this answers is not in the document]
Comment (espzav) [reply examples/noglib/main/noglib_main.c]: I agree 👍

**Review by espzav — APPROVED** [review]

**Pull request merged**